sites/svelte-5-preview/src/routes/docs/content/01-api/05-imports.md

```diff
@@ -59,6 +61,8 @@ const app = hydrate(App, {
 });
 ```
 
+As with `mount`, effects will not run during `hydrate` — use `flushSync()` immediately afterwards if you need them to.
```

Comment:
can you mirror that into the docs at `documentation/...`? because those are the soon-to-be final docs